packages/material-ui/src/LinearProgress/LinearProgress.d.ts

```diff
@@ -7,8 +7,13 @@ export interface LinearProgressProps
   value?: number;
   valueBuffer?: number;
   variant?: 'determinate' | 'indeterminate' | 'buffer' | 'query';
+  styles?: { [P in LinearProgressStylesKeys]?: React.CSSProperties };
 }
 
+export type LinearProgressStylesKeys =
```

Comment:
For consistency with the `*ClassKey` convention, I would call this `LinearProgressStyleKey`. It shouldn't be plural because it's a type; think of using it in an _is-a_ sentence:

"`bar1` is a `LinearProgressStyleKey`".

(that's my type naming philosophy anyway 🙂).

Reply:
I agree with this convention.